packages/action-text-field/action-text-field.scss

```diff
@@ -0,0 +1,27 @@
+@import "~@fremtind/jkl-core/variables/_all.scss";
+
+.jkl-text-field {
+    &__input {
+        padding-right: 3rem;
```

Comment:
Might be a good idea to put these into a variable, makes it look a bit more deliberate